pycascadia/remove_restore.py

```diff
@@ -65,6 +65,10 @@ def calc_diff_grid(base_grid, update_grid, diff_threshold=0.0, window_width=None
         print("Update grid is entirely outside region of interest. Skipping.")
         return None
 
+    if all_values_are_nodata(update_grid.grid):
+        print("Update grid consists entirely of no_data_values. Skipping")
```

Comment:
Just for consistency with previous messages.

```suggestion
        print("Update grid consists entirely of no_data_values. Skipping.")
```